packages/frontend/amp/components/SubMeta.tsx

```diff
@@ -155,10 +172,24 @@ export const SubMeta: React.FC<{
 
     return (
         <>
-            <div className={guardianLines}>
+            <div
+                className={composeLabsCSS(
```

Comment:
Re: whether we can avoid some of these composeLabsCSS calls, perhaps we could make these cases with grey lines accept a pillar and then add a new pillar colour field called e.g. neutral which can hold the grey palette colour for each pillar (which would be the same for all other than labs). Does that make sense? The advantage would be to remove the special labs handling/things become just pillars again.

Reply:
Rolled out this change. We now only need to use `composeLabsCSS` in two files - Text and Standfirst components where we need to switch to Sans serif font. The 'neutral' prop in palette resolved every other case 👍